### PR TITLE
improvements and bugfixes

### PR DESCRIPTION
**Pull request by petrparolek**
- unify app structure and added .editorconfig
- fixed sessions

### CODE REVIEW

File: config/common.neon [hunk 1]
```diff
@@ -17,4 +17,4 @@ dibi:
 	database: playground
```
Comment (f3l1x) [inline config/common.neon:17]: Jeste prosim common.neon prejmenovat na config.neon.

Comment (petrparolek) [reply config/common.neon:17]: opraveno

**Comment by petrparolek** [comment]
Is it ok?

**Comment by paveljanda** [comment]
Sorry but I do not agree with this directory structure. As far as I know this project will be used by newbies. Also, nette/web-project (which is meant as an newbie starting line) uses the previous file structure. I don't our skeleton apps to use some custom directory structure when all the world around us uses something else.

**Comment by paveljanda** [comment]
Reverting changes

**Comment by f3l1x** [comment]
Ok @paveljanda no problem.

**Comment by paveljanda** [comment]
@f3l1x Thanks for understanding. 👍

**Pull request merged**